index.html

```diff
@@ -1 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+    <title>Apartment Owners Association</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./assets/css/styles.css">
+</head>
+<body>
+<header class="header">
+    <div class="container-lg flex align-items-center full-height">
+        <img src="./assets/images/logo.svg" height="40" width="40" alt="logo">
+        <p class="company-name text-size-sx font-bold uppercase">Apartment Owners Association</p>
+        <div class="hamburger">
+            <span></span>
+            <span></span>
+            <span></span>
+        </div>
+        <nav class="navigation uppercase">
+            <ul>
+                <li><a href="#">About us</a></li>
+                <li class="dropdown-menu"><a href="#">membership
+                    <svg class="dropdown-menu-icon" width="14" height="9" viewBox="0 0 14 9" fill="none"
```

Comment:
Try avoiding overwhelming the DOM with SVG elements.
In general, adding SVG inline is a common practice, but this refers to content elements, not decorations. E.g. having 100 menu items there will be (100 x number_of_svg_elements) amount of additional DOM elements. 
Add SVG as a background image of a pseudo element. Actually pseudo elements are not in DOM, thus good for overall performance.